Gopkg.toml

```diff
@@ -3,8 +3,8 @@
   version = "1.0.5"
 
 [[constraint]]
-  branch = "master"
```

Comment:
You should keep the branch, we are not pining this way by revision.

Reply:
Yes I tried it but there are many changes in drone go lib and this plugin is not compatible with the latest drone go lib.